app/routes.php

```diff
@@ -134,4 +138,5 @@
     $app->post('/api/sendEmail', SendEmailController::class);
     $app->get('/api/lockApplicantField', LockApplicantFieldController::class);
     $app->get('/api/getAptitudeScore', GetApplicantAptitudeController::class);
+    $app->get('/api/calendar', GetAllCalendarEventsController::class);
```

Comment:
not sure about the route url here, its not descriptive enough. look at the other routes, they are all obvious what they do